Subject: util_path: Add description for util_path_exists()
From: Jan Hoeppner <jan.hoeppner@de.ibm.com>

Summary:     zpcictl: Add tool to manage PCI devices
Description: Use the zpcictl tool to manage PCI devices on the IBM Z
             platform.  Initial functions include generating firmware
             error logs, resetting PCI devices, and preparing a device
             for further repair actions.
Upstream-ID: d0e2caf0ffb195568bba89a95549a5a4f026a4e6
Problem-ID:  RAS1703

Upstream-Description:

             util_path: Add description for util_path_exists()

             Signed-off-by: Michael Holzheu <holzheu@linux.vnet.ibm.com>


Signed-off-by: Jan Hoeppner <jan.hoeppner@de.ibm.com>
---
 libutil/util_path.c |   11 +++++++++++
 1 file changed, 11 insertions(+)

--- a/libutil/util_path.c
+++ b/libutil/util_path.c
@@ -195,6 +195,17 @@ free_str:
 	return rc;
 }
 
+/**
+ * Test if path to directory or file exists
+ *
+ * This function has the same semantics as "-e path" in bash.
+ *
+ * @param[in] fmt   Format string for path to test
+ * @param[in] ...   Variable arguments for format string
+ *
+ * @returns   true  Path exists
+ *            false Otherwise
+ */
 bool util_path_exists(const char *fmt, ...)
 {
 	va_list ap;
